ui/src/App.tsx

```diff
@@ -28,6 +28,17 @@ function App() {
                 rpc: ['http://localhost:26657'],
               },
             },
+            {
+              testChain: {
+                chainId: 'agoric-emerynet-8',
+                chainName: 'emerynet',
+                iconUrl: 'agoric.svg',
+              },
+              apis: {
+                rest: ['https://emerynet.api.agoric.net'],
+                rpc: ['https://emerynet.rpc.agoric.net'],
+              },
```

Comment:
I ended up using the network config to do some codegen for deploying the contract:

https://github.com/Agoric/dapp-orchestration-basics/pull/61/commits/9fe5ab90a66da58f8f27edfd041fed8cc4b6afb8

I could write a .json file or a .js module with chainId and rest / rpc, if it seems useful. Maybe more trouble than it's worth.

Reply:
Seems potentially useful, I guess the output would need to be checked in somewhere though, might as well just keep this here.